cumulusci/core/tests/cassettes/TestGithub.test_validate_no_repo_exc.yaml

```diff
@@ -0,0 +1,55 @@
+interactions:
+    - request:
+          body: null
+          headers:
+              Request-Headers:
+                  - Elided
+          method: GET
+          uri: https://api.github.com/user
+      response:
+          body:
+              string: '{"login":"e2ac67","id":91303375,"node_id":"MDQ6VXNlcjkxMzAzMzc1","avatar_url":"https://avatars.githubusercontent.com/u/91303375?v=4","gravatar_id":"","url":"https://api.github.com/users/e2ac67","html_url":"https://github.com/e2ac67","followers_url":"https://api.github.com/users/e2ac67/followers","following_url":"https://api.github.com/users/e2ac67/following{/other_user}","gists_url":"https://api.github.com/users/e2ac67/gists{/gist_id}","starred_url":"https://api.github.com/users/e2ac67/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/e2ac67/subscriptions","organizations_url":"https://api.github.com/users/e2ac67/orgs","repos_url":"https://api.github.com/users/e2ac67/repos","events_url":"https://api.github.com/users/e2ac67/events{/privacy}","received_events_url":"https://api.github.com/users/e2ac67/received_events","type":"User","site_admin":false,"name":null,"company":null,"blog":"","location":null,"email":null,"hireable":null,"bio":null,"twitter_username":null,"public_repos":0,"public_gists":0,"followers":0,"following":0,"created_at":"2021-09-24T03:53:02Z","updated_at":"2021-09-24T03:59:40Z"}'
+          headers:
+              Content-Type:
+                  - application/json; charset=utf-8
+              Others: Elided
+          status:
+              code: 200
+              message: OK
+    - request:
+          body: null
+          headers:
+              Request-Headers:
+                  - Elided
+          method: GET
+          uri: https://api.github.com/user/orgs?per_page=100
+      response:
+          body:
+              string: "[]"
+          headers:
+              Content-Type:
+                  - application/json; charset=utf-8
+              Others: Elided
+          status:
+              code: 200
+              message: OK
+    - request:
+          body: null
+          headers:
+              Request-Headers:
+                  - Elided
+          method: GET
+          uri: https://api.github.com/user/repos?per_page=100
+      response:
+          body:
+              string: "[]"
+          headers:
+              Content-Type:
+                  - application/json; charset=utf-8
+              X-OAuth-Scopes:
+                  - gist, repo
+              Others: Elided
```

Comment:
@prescod Regarding our conversation last week, this is the use case I was describing. I'd like sf_vcr to pass any headers I've added myself to the through to the response. Looks like `headers` is getting ignored in its entirety right now.